packages/ozone/src/api/util.ts

```diff
@@ -30,17 +30,28 @@ import { ids } from '../lexicon/lexicons'
 
 export const getPdsAccountInfo = async (
   ctx: AppContext,
-  did: string,
-): Promise<AccountView | null> => {
+  did: string | string[],
```

Comment:
I'm inclined to just have this always take a `string[]` if it's always returning a `Map` of account views. Just keeps the function a bit simpler